EFAuditer/EFAuditServiceExtensions.cs

```diff
@@ -12,6 +13,15 @@ namespace EFAuditer
 {
     public static class EFAuditServiceExtensions
     {
+        private static readonly JsonSerializerOptions JsonSettings = BuildJsonSettings();
+
+        private static JsonSerializerOptions BuildJsonSettings()
+        {
+            var settings = Audit.Core.Configuration.JsonSettings;
+            settings.Converters.Add(new JsonStringEnumConverter());

```

Comment:
Shouldn't we do this during startup?

Reply:
It will mean we'll have to refactor the test, but the trouble with this is that we're changing the `JsonSettings` during the running of the application, which is very strange indeed!

Reply:
I'm not sure how we'd do it and then pass those settings to the auditer. If you do something like this: 

`Audit.Core.Configuration.JsonSettings.Converters.Add(new JsonStringEnumConverter());`

in startup it doesn't actually add a converter to the default settings. 
We could pass the settings with the converter added into the AuditAction method from AddEFAuditer ?

Reply:
That's very strange! Do you have any idea why?

Reply:
I've sorted it now, I think where I had put it wasn't being run by startup. 
The issue now is refactoring tests so that startup is run before the EF auditer tests

Reply:
You could commonise that piece of logic (e.g. put it in an extension method) and then call it in the test setup.

It's not an ideal solution, but it would be quick.

Reply:
The issue with that approach is that it means it's called before every test, and if you try edit the serializer settings when you've already audited something it throws an error (so the first test passes and all others fail).

Reply:
Used a class fixture which sets the json settings for the test class to fix this.

Reply:
You could also have got around this by making the setup method idempotent - i.e. only add the the converter if it's not already in the list of converters.